presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -94,7 +97,20 @@ public void testNativeExecutionWithProjection()
         PrestoSparkQueryRunner queryRunner = (PrestoSparkQueryRunner) getQueryRunner();
         queryRunner.resetSparkContext(getNativeExecutionShuffleConfigs());
         try {
-            assertQuerySucceeds(session, "SELECT * FROM test_order");
+            MaterializedResult actual = queryRunner.execute(session, "SELECT * FROM test_order");
```

Comment:
Can we use all-Java-Presto query runner as a comparison point? See presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java